data/settings.js

```diff
@@ -7329,6 +7329,20 @@ passdb ldap {
 \`\`\``
 	},
 
+	passdb_fields_import_all: {
+		tags: [ 'passdb' ],
+		values: setting_types.BOOLEAN,
+		default: 'yes',
+		seealso: [ 'passdb_fields' ],
+		text: `
+If yes, import all fields returned by the passdb lookup. If no, require
+[[setting,passdb_fields]] to explicitly add wanted fields.
+
+::: info
+This setting's default value for \`passdb ldap\` is \`no\`.
+:::`
```

Comment:
I opened discussions in part IV about these kind of info-boxes about defaults. Would be nice to find a better way.